tests/baselines/reference/config/initTSConfig/Default initialized TSConfig/tsconfig.json

```diff
@@ -57,7 +58,6 @@
     // "downlevelIteration": true,                       /* Emit more compliant, but verbose and less performant JavaScript for iteration. */
     // "sourceRoot": "",                                 /* Specify the root path for debuggers to find the reference source code. */
     // "mapRoot": "",                                    /* Specify the location where debugger should locate map files instead of generated locations. */
-    // "inlineSourceMap": true,                          /* Include sourcemap files inside the emitted JavaScript. */
     // "inlineSources": true,                            /* Include source code in the sourcemaps inside the emitted JavaScript. */
```

Comment:
Should these all be getting moved up? I was hoping we'd end up not moving any of them, but it does seem like the source map should be synced.

Reply:
we don't move the categories and their order is fixed but having to manage order inside categories is more work without much benefit in my opinion.

And i could move `inlineSources`, `sourceRoot`, `mapRoot` but as first before outFile but thats all i could do for this one.
Having said that, personally I think we are in good spot as is.. The ones enabling different file level emits are grouped and then the other emit stuff and `sourceMap` related config options are grouped separately.